### ISSUE TITLE
Feature request: editable email templates

### ISSUE THREAD

**Issue by freelock** [ledgersmb/LedgerSMB#2489]
When sending invoices, LSMB already supports using placeholder tokens for things like invoice number, amount, etc. It would be great to have a standard invoice message that would auto-populate the body field of an email form, and could then be edited before sending.

**Comment by sbts** [comment]
Fixing this would also resolve #2488 
Mentioning it here so issues are linked

**Comment by sbts** [comment]
As we already have some sort of templating in use here, I'd propose we store both the body in the db as part of the existing output template arrangement (as we do for orders, invoices etc)
That should immediately provide the desired mechanisms for editing etc

For the subject we should simply provision for the subject being included as the first line in the body in the form of a line like this
`SUBJECT: the desired subject goes here`
That seems to be the standard way to pass a subject line to most MTA's so may not even need us to do anything special to make it work right

**Comment by freelock** [comment]
Just to comment, fixing this would not resolve #2488 -- that issue is not a matter of a template -- that issue is that when you change the text in an existing input, it is entirely ignored. This issue is a feature request for new functionality...

**Comment by sbts** [comment]
@freelock 
Agreed, fixing this doesn't resolve #2488 I only mentioned it do ensure it doesn't get missed if the templating side is resolved.

**Comment by ehuelsmann** [comment]
The existing templating functionality comes from recurring email which comes with the option to set a message on the mail body
